docs/manual/notifications/new-interface.md

```diff
@@ -0,0 +1,13 @@
+# Notification Interface V2 (After v3.3.4)
```

Comment:
We don't typically mention version numbers in this context.  We just present it as the current interface so the title would be "Notification Interface" or something that describes this product area.

Reply:
Updated!